app/scrapers/twitter.py

```diff
@@ -0,0 +1,24 @@
+import requests
+
+
+class Twitter():
+    """docstring for Twitter"""
```

Comment:
Something a bit more explanatory here?  "This scraper takes a query and a count and returns the results of a Twitter search which is executed via the Loklak API."

Reply:
I have added an explanation near the Search function which amounts to the same thing, but yeah sure I'll include this as well 😄